docs/cases/cases-overview.asciidoc

```diff
@@ -102,3 +102,45 @@ NOTE: Once a visualization has been added to a case, it cannot be modified or de
   
 [role="screenshot"]
 image::images/cases-open-vis.png[Shows where the Open Visualization option is]
+
+[float]
+[[cases-export-import]]
+== Export and import cases
+
+Cases can be <<cases-export, exported>> and <<cases-import, imported>> as saved objects through the Kibana {kibana-ref}/managing-saved-objects.html[Saved Objects] UI.
+
+IMPORTANT: Before importing a Lens visualization into a space, ensure the visualization's data is present. Without it, the Lens visualization won't work.
+
+[float]
+[[cases-export]]
+=== Export a case
+Use the export option to move cases between different Kibana instances. When you export a case, the case details, user actions, and comments are exported to a newline-delimited JSON (NDJSON) file. During the export process, text string comments, case alerts, and lens visualizations are exported.
```

Comment:
The last two sentences have two separate lists that both say what's included when you export a case (and "comments" appear twice). Could we combine these lists somehow?
```suggestion
Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (NDJSON) file: case details, user actions, text string comments, case alerts, and lens visualizations.
```